reconcile.py

```diff
@@ -78,6 +75,11 @@ def get_new_tracking_pos_costs_maps(
     ) for (k, v) in group_trackings_to_po.items()})
     po_to_cost_map.update(group_po_to_cost)
 
+  non_portal_reimbursements_retriever = NonPortalReimbursementsRetriever(config)
+  non_portal_reimbursements = non_portal_reimbursements_retriever.non_portal_reimbursements
+  trackings_to_costs_map.update(non_portal_reimbursements.trackings_to_costs)
```

Comment:
This seems like it does the wrong thing if the tracking # in question is already in the `trackings_to_costs_map` -- it will silently be overridden.  We should either error out if a tracking is reimbursed both in a portal and not in a portal, or sum the two, depending on how we intend to use this functionality.

Reply:
I think an error is probably more appropriate then